src/components/subscriptions/SubscriptionManagementPage.jsx

```diff
@@ -1,19 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Helmet } from 'react-helmet';
 
 import Hero from '../Hero';
 import SearchBar from '../SearchBar';
 import SubscriptionData, { SubscriptionConsumer } from './SubscriptionData';
 import SubscriptionDetails from './SubscriptionDetails';
-import AddUsersDropdown from './AddUsersDropdown';
 import LicenseAllocationNavigation from './LicenseAllocationNavigation';
 import TabContentTable from './TabContentTable';
+import StatusAlert from '../StatusAlert';
 
 import './styles/SubscriptionManagementPage.scss';
+import AddUserButton from './AddUsersButton';
 
 const PAGE_TITLE = 'Subscription Management';
 
 export default function SubscriptionManagementPage() {
+  const [status, setStatus] = useState({
```

Comment:
This can be defined at https://github.com/edx/frontend-app-admin-portal/blob/2a12ed5dee4e35e63b7ff9bade0fef7a51edec44/src/components/subscriptions/SubscriptionData.jsx#L20